db/config.cc

```diff
@@ -1474,6 +1502,11 @@ std::istream& operator>>(std::istream& is, error_injection_at_startup& eias) {
     return is;
 }
 
+
+std::istream& operator>>(std::istream& is, object_storage_endpoint_param& f) {
+    return is;
+}
```

Comment:
This operator should have non-empty implementation, if in use, or not exist at all.

Reply:
I couldn't figure out what it is used for, any idea why we have it there? Obviously if you remove it, you get a huge pile of compile errors.

Reply:
I see `db::seed_provider_type` has this comment:
```
std::istream& operator>>(std::istream& is, db::seed_provider_type& s) {
    // FIXME -- this operator is used, in particular, by boost lexical_cast<>
    // it's here just to make the code compile, but it's not yet called for real
    throw std::runtime_error("reading seed_provider_type from istream is not implemented");
    return is;
}
```
are you ok with throwing here as well?

Reply:
@xemul ping

Reply:
np, updated, thanks!